director/template/vars_file_arg.go

```diff
@@ -29,6 +39,18 @@ func (a *VarsFileArg) UnmarshalFlag(filePath string) error {
 		return bosherr.WrapErrorf(err, "Deserializing variables file '%s'", filePath)
 	}
 
+	if fileScope != "" {
+		scopedVars := map[interface{}]interface{}{}
```

Comment:
Can also change StaticVariables to be `map[interface{}]interface{}` (https://play.golang.org/p/sVn3QLK3FX) and then nest them, instead of making a map.